src/kbmod_wf/resource_configs/klone_configuration.py

```diff
@@ -8,8 +8,8 @@
 walltimes = {
     "compute_bigmem": "01:00:00",
     "large_mem": "04:00:00",
-    "sharded_reproject": "04:00:00",
-    "gpu_max": "08:00:00",
+    "sharded_reproject": "01:00:00",
+    "gpu_max": "01:00:00",
```

Comment:
I reduced the time requested for each of these. @DinoBektesevic I think that 1hr should generally be enough to finish a search, but let me know if this should be pushed back up.